ccd-definition/CaseEventToFields/DefendantResponseAdmitPartLRspec.json

```diff
@@ -377,9 +377,9 @@
     "PageID": "WhyDoesNotPayImmediately",
     "PageDisplayOrder": 21,
     "PageFieldDisplayOrder": 1,
-    "PageShowCondition": "respondent1ClaimResponseTypeForSpec=\"PART_ADMISSION\" AND defenceAdmitPartPaymentTimeRouteRequired !=\"IMMEDIATELY\" AND specDefenceAdmittedRequired = \"No\"",
+    "PageShowCondition": "respondent1ClaimResponseTypeForSpec!=\"FULL_DEFENCE\" AND respondent1ClaimResponseTypeForSpec!=\"COUNTER_CLAIM\" AND defenceAdmitPartPaymentTimeRouteRequired !=\"IMMEDIATELY\" AND specDefenceAdmittedRequired = \"No\"",
```

Comment:
Please, check that the "Why does not pay immediately" page also appears for full admission, because specDefenceAdmittedRequired is only in part admission, if I'm not mistaken

Reply:
I hade to remove that "specDefenceAdmittedRequired" part of the condition and now I can see that page on full admission. Thanks for pointing that out.